package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
Hrmmm...  I know we don't like copying code, but we really need to have the loader not have any external dependencies.  I think either we need code that provides "extra features" in the presence of other packages but not have a hard dependency, or we need to narrow the code to just meet the requirement, even though it wouldn't be the sort of API we would normally implement.